config/dev/services.yml

```diff
@@ -0,0 +1,12 @@
+imports:
+  - { resource: '../prod/services.yml' }
+
+services:
```

Comment:
I'm not sure I understand the purpose of this bifurcation. It seems like you're inheriting a default set of prod services, and then overriding them for dev?

I think we build the phar in production mode, so if you need BufferedOutput or something else from this file for formatting to work, it won't work with the phar.

Reply:
This is only used during testing. It’s necessary because during testing, we used buffered output instead of the console output. The same issue occurs for input.

Without this, services are auto wired incorrectly during testing.

Reply:
> so if you need BufferedOutput or something else from this file for formatting to work, it won't work with the phar.

BufferedOutput is not required for formatting. It's required for capturing output during testing.